### ISSUE TITLE
`--dry-run` does Zabbix interaction anyway

### ISSUE THREAD

**Issue by lroellin** [zabbix-tooling/zabbix-ldap-sync#4]
I'm not sure how official that parameter is yet, since it's only in the usage but not in the README.

Anyway, it doesn't seem to have an effect for me. It created the groups happily anyway :)

Is this expected?

**Comment by lroellin** [comment]
Having had a look at it, I think it's not used anywhere. Can you confirm that?

**Comment by scoopex** [comment]
No :-)

It ssems that this is not possible :-)

```
    def create_missing_groups(self):
        """
        Creates any missing LDAP groups in Zabbix

        """
        missing_groups = set(self.ldap_groups) - set([g['name'] for g in self.get_groups()])

        for eachGroup in missing_groups:
            self.logger.info('Creating Zabbix group %s' % eachGroup)
            if not self.dryrun:
                grpid = self.create_group(eachGroup)
                self.logger.info('Group %s created with groupid %s' % (eachGroup, grpid))
```

**Comment by scoopex** [comment]
Fixed with pull request #10
